Add new pickup and delivery fees (#226)
Progressus-io/postnl-for-woocommerce: readme.txt

== Changelog ==

= 5.8.1 (2025-09-06) =

> * Add: Separate fees for Home delivery and Pick-up point delivery.
> * Tweak: Show fee amounts on checkout tabs.

Reviewer: ```suggestion
* Add: Option to apply extra fees for Home Delivery and Pickup Point Delivery.
```

And lets add it to `### 5.8.0 (2025-xx-xx) =` section since its not released yet
